tests/src/common/Wsk.scala

```diff
@@ -252,13 +252,13 @@ class WskAction()
      */
     def invoke(
         name: String,
-        parameters: Map[String, JsValue] = Map(),
+        parameters: Map[String, String] = Map(),
```

Comment:
I see. To make that more clear, we should create tests that pass properly formatted JSON strings to the CLI. The existing automation only passes words enclosed in quotes to the CLI, which is the reason I changed the JsValue type to String.